cli/src/stake.rs

```diff
@@ -153,52 +156,62 @@ impl StakeSubCommands for App<'_, '_> {
                         .takes_value(true)
                         .validator(is_amount_or_all)
                         .required(true)
-                        .help("The amount to send to the stake account, in SOL; accepts keyword ALL")
+                        .help(
+                            "The amount to send to the stake account, in SOL; accepts keyword ALL",
+                        ),
                 )
-                .arg(
-                    pubkey!(Arg::with_name("custodian")
+                .arg(pubkey!(
+                    Arg::with_name("custodian")
                         .long("custodian")
                         .value_name("PUBKEY"),
-                        "Authority to modify lockups. ")
-                )
+                    "Authority to modify lockups. "
```

Comment:
The trailing period with space after it doesn't quite seem to line up with what the macro drops in.
https://github.com/solana-labs/solana/blob/d84dcd37bcb6afceb533a3883ec05e048a190903/cli/src/lib.rs#L2-L11

We seemingly don't need the trailing space or the period. Ie, running `solana create-stake-account --help`, I see:
```
        --custodian <PUBKEY>
            Authority to modify lockups. , one of:
              * a base58-encoded public key
              * a path to a keypair file
              * a hyphen; signals a JSON-encoded keypair on stdin
              * the 'ASK' keyword; to recover a keypair via its seed phrase
              * a hardware wallet keypair URL (i.e. usb://ledger)
```
There are a bunch of these so thinking we defer to another PR, but figured I'd mention it as I just saw it. Probably worth auditing all instances of `pubkey!(` to check this too